tools/Status/Status.py

```diff
@@ -154,16 +154,9 @@ def _state_order(state):
         return None
 
 
-def _format(field: str, value: Any) -> str:
+def _format(field: str, value: Any, state_styles: dict) -> str:
     if field == "State":
-        style = {
-            "RUNNING": "[blue]",
-            "COMPLETED": "[green]",
-            "PENDING": "[magenta]",
-            "FAILED": "[red]",
-            "TIMEOUT": "[red]",
-        }
-        return style.get(value, "") + str(value)
+        return state_styles.get(value, "") + str(value)
```

Comment:
do this:

```py
style = {
  # ... (same as before)
}
style.update(state_styles)
```

Then you don't have to change any other code, only pass the `state_styles` from the options through.